deploy/README.md

```diff
@@ -493,12 +615,11 @@ ebpf:
 
 Install `falco` on `GKE` using Helm:
```

Comment:
Same as above comment:

Since the user isn't actually using helm to install, let's change the wording here to something more along the lines of "Install `falco` by generating the yaml files using Helm" or something like that

but also let's get rid of the "on `GKE`" since these steps should work for both GKE and non-GKE here